components/Sidebar.tsx

```diff
@@ -312,10 +324,32 @@ export const DocsNav = () => {
           <DocLink uri='/specification-links' label='Specification Links' />
         </div>
       </div>
+      {/* Resources */}
+      <div className='mb-2 bg-slate-200 p-2 rounded'>
+        <div className='flex justify-between w-full items-center' onClick={handleClickResources}>
+          <div className='flex  items-center align-middle'>
+            <img src='/icons/clipboard.svg' alt='eye icon' className='mr-2' />
+            <SegmentHeadline label='Resources' />
+          </div>
+          <svg id='arrow' className='arrow' style={{ transform: rotateResources, transition: 'all 0.2s linear' }} xmlns='http://www.w3.org/2000/svg' fill='none' height='32' viewBox='0 0 24 24' width='24'><path clipRule='evenodd' d='m16.5303 8.96967c.2929.29289.2929.76777 0 1.06063l-4 4c-.2929.2929-.7677.2929-1.0606 0l-4.00003-4c-.29289-.29286-.29289-.76774 0-1.06063s.76777-.29289 1.06066 0l3.46967 3.46963 3.4697-3.46963c.2929-.29289.7677-.29289 1.0606 0z' fill='#707070' fillRule='evenodd' /></svg>
+
+        </div>
+        <div
+          className={classnames( 'ml-6', { 'hidden': !active.getResources })}
+          id='resources'
+        >
+          <DocLink uri='/resources/books' label='Books' />
```

Comment:
These are static links. I'd like to avoid this. Instead I'd like you to go through the data and extract the resource categories available there to build dynamically these links.